packages/gatsby/static/configuration/yarnrc.json

```diff
@@ -414,7 +414,7 @@
       "description": "Defines what linker should be used for installing Node packages (useful to enable the node-modules plugin), one of: `pnp`, `pnpm` and `node-modules`.",
       "type": "string",
       "default": "pnp",
-      "examples": ["pnp", "pnpm", "node-modules"]
+      "enum": ["pnp", "pnpm", "node-modules"]
```

Comment:
It isn't an enum of just these three values, plugins can add more options linkers

Reply:
For the purpose of the JSON schema (which only enhances the IDE experience) it's probably fine to define them as an enum. Users can still set them to other values if they need, it'll just be displayed with yellow squiggles.

Reply:
Didn't know `nodeLinker` can be other than "pnp", "pnpm", "node-modules" (documentation says "one of: `pnp`, `pnpm` and `node-modules`").
Fixed! Now it should accept any string value and autocomplete will show known values: "pnp", "pnpm", "node-modules".

Reply:
The solution is taken from https://github.com/json-schema-org/json-schema-spec/issues/665

Reply:
Perfect, could you update `pnpMode` while you're at it?

> which only enhances the IDE experience

Note that VSCode already shows these three values as auto-complete options

Reply:
@merceyz As I can see, `pnpMode` is already `"enum": ["strict", "loose"]` and other values are disallowed, IIUC. 

>Note that VSCode already shows these three values as auto-complete options

Maybe you're right and `examples` should be auto-completed, but it feels a bit wrong to complete an arbitrary example in general case. In this particular case, auto-completing `examples` is correct.

Reply:
> As I can see, pnpMode is already "enum": ["strict", "loose"] and other values are disallowed, IIUC.

Plugins can provide their own value here as well so it should follow the same config as `nodeLinker`